src/common/roll_renderer.js

```diff
@@ -580,9 +580,8 @@ class Beyond20RollRenderer {
             }
 
             // If rolling the attack, check for critical hit, otherwise, use argument.
-            if (request.rollAttack) {
-                if (to_hit.length > 0)
-                    this.processToHitAdvantage(to_hit_advantage, to_hit)
+            if (request.rollAttack && to_hit.length > 0) {
```

Comment:
hummm, interesting, I assume you found a bug here we hadn't noticed before? 
Anyways, `to_hit` might be `undefined` if there wasn't one, so the `.length > 0` would fail. doing `&& !!to_hit` would be better I think or `&& to_hit && to_hit.length > 0)`